packages/esm-patient-conditions-app/src/conditions/conditions-form.component.tsx

```diff
@@ -64,7 +77,7 @@ const ConditionsForm: React.FC<ConditionFormProps> = ({ closeWorkspace, conditio
       <Form className={styles.form} onSubmit={methods.handleSubmit(onSubmit, onError)}>
         <ConditionsWidget
           patientUuid={patientUuid}
-          closeWorkspace={closeWorkspace}
+          discardChangesAndCloseWorkspace={discardChangesAndCloseWorkspace}
```

Comment:
Nit: it's usually a bit cleaner to name the function prop with the `on` prefix, e.g. `onDiscardChangesAndCloseWorkspace` here.